yt_dlp/extractor/livestreamfails.py

```diff
@@ -0,0 +1,50 @@
+from .common import InfoExtractor
+from ..utils import traverse_obj
+import json
+import time
+import calendar
+
+
+class LivestreamfailsIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?livestreamfails\.com/clip/(?P<id>[0-9]+)'
+    _TESTS = [{
+        'url': 'https://livestreamfails.com/clip/139200',
+        'md5': '8a03aea1a46e94a05af6410337463102',
+        'info_dict': {
+            'id': '139200',
+            'ext': 'mp4',
+            'display_id': 'ConcernedLitigiousSalmonPeteZaroll-O8yo9W2L8OZEKhV2',
```

Comment:
don't see how this is useful

Reply:
`id` and `ext` are from the [example](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-support-for-a-new-site). Should I remove them?

And as for the `display_id`: Twitch has its own IDs for clips (used like `https://clips.twitch.tv/ConcernedLitigiousSalmonPeteZaroll-O8yo9W2L8OZEKhV2`). I figured I should include that since the regular ID is only valid for livestreamfails.com. Should I not include the Twitch ID?